samples/cellular/http_update/application_update/boards/nrf9160dk_nrf9160_ns.conf

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
Missing a line with `#` at the top of the file. Same for `boards/nrf9161dk_nrf9161_ns.conf`.

Apart from that, looks good.

Reply:
updated